addons/gordon_reskins/rfdeagle/config.cpp

```diff
@@ -0,0 +1,37 @@
+class CfgPatches
+{
+	class gordon_reskins_rfdeagle
+	{
+		units[] = {};
+		weapons[]=
+		{
+			"FP_GOR_hgun_DEagle_chrome_RF"
+		};
+		requiredVersion = 0.1;
+		requiredAddons[]=
```

Comment:
please add `skipWhenMissingDependencies = 1;` here, in case the CDLC is not loaded anymore

Reply:
good call, will do right now